DXMainClient/DXGUI/Multiplayer/CnCNet/GameCreationWindow.cs

```diff
@@ -82,10 +88,30 @@ public override void Initialize()
                 UIDesignConstants.CONTROL_HORIZONTAL_MARGIN, ddMaxPlayers.Y + 1, 0, 0);
             lblMaxPlayers.Text = "Maximum number of players:".L10N("Client:Main:GameMaxPlayerCount");
 
+            // Difficulty selector
+            ddGameDifficulty = new XNAClientDropDown(WindowManager);
+            ddGameDifficulty.Name = nameof(ddGameDifficulty);
+            ddGameDifficulty.ClientRectangle = new Rectangle(tbGameName.X, ddMaxPlayers.Bottom + 20,
+                tbGameName.Width, 21);
+
+            foreach (string difficulty in OnlineGameDifficultyOptions)
+            {
+                string localizedDifficulty = difficulty.L10N($"INI:ClientDefinitions:Difficulty:{difficulty}");
```

Comment:
Are you sure using the difficulty string instead of the index as the key name? The difficulty string might contains unwanted characters like `;()=:`